acceptance_modelisation/toolbox.py

```diff
@@ -27,7 +29,54 @@ def compute_rotation_speed_fov(time_evaluation: Time,
     """
     pointing_altaz = pointing_sky.transform_to(AltAz(obstime=time_evaluation,
                                                      location=observatory_earth_location))
-    omega_earth = OMEGA_EARTH*u.rad
+    omega_earth = OMEGA_EARTH * u.rad
     omega = omega_earth * np.cos(observatory_earth_location.lat) * np.cos(pointing_altaz.az) / np.cos(
         pointing_altaz.alt)
-    return omega
\ No newline at end of file
+    return omega
+
+
+def get_unique_wobble_pointings(observations: Observations, max_angular_separation=0.4):
+    """
+    Compute the angular separation between pointings and return a list
+    of detected wobbles with their associated similar pointings
+
+    Parameters
+    ----------
+    observations : gammapy.data.observations.Observations
+        The list of observations
+    max_angular_separation : float
+        The maximum angular separation between a wobble position and associated runs, in degrees
+
+    Returns
+    -------
+    unique_wobble_list : `numpy.array`
+        Array of wobble name associated with each run
+    """
+    all_ra_observations = np.array([obs.get_pointing_icrs(obs.tmid).ra.to_value(u.deg) for obs in observations])
```

Comment:
Could be directly set as astropy quantity by multiplying by u.deg. Would make the code after a bit less dense by avoiding to reset units each time.

Reply:
Done.
I had to change the logging.info output to be able to round the printed values